### DISCUSSION TITLE
Can't Gain Ranked Score

### DISCUSSION THREAD

**Discussion by SuperDoesGithub** [ppy/osu#24416] [Q&A] [answered]
I've been playing Osu! for about 4 days now and I still have 0 ranked score. I mainly play ranked maps and biased on what I've seen about it that should give me ranked score. But it's not no madder how good I do.

**Comment by monochrome22** [comment, marked as answer]
Answered here: https://github.com/ppy/osu/discussions/22838#discussioncomment-5299551.
Ranked score is not yet implemented for lazer.
